cfme/tests/v2v/test_v2v_copy.py

```diff
@@ -0,0 +1,115 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from widgetastic.utils import partial_match
+
+from cfme.utils.log import logger
+
+pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+
+
+def test_basic_navigation(appliance):
+    """Test takes an appliance and tries to navigate and fill the form for Infra Map and Migration Plan.
+
+    This is a dummy test just to check navigation.
+    # TODO : Replace this test with actual test.
+    Expected form_data format:
+    form_data = {
+        'general':
+            {
+                'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                'description': fauxfactory.gen_string("alphanumeric", length=50)
+            },
+        'cluster':
+            {
+                'mappings': [
+                    {
+                        'sources': ['source_Datacenter_name \ source_Cluster_name'],
```

Comment:
much better with the generic values here!

I think you can remove the 2nd item in the lists for datastore and network, doesn't need to be that verbose